test/parallel/test-whatwg-url-tojson.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const URL = require('url').URL;
+
+// Tests below are not from WPT.
+const url = new URL('http://example.org');
```

Comment:
Can you port the [WPT tests](https://github.com/w3c/web-platform-tests/blob/master/url/url-tojson.html) here like e.g. what `test-whatwg-url-searchparams-set.js` does for `urlsearchparams-set.html`? Thanks!